### DISCUSSION TITLE
Conflict with ESLint, enable on demand

### DISCUSSION THREAD

**Discussion by ricardobeat** [biomejs/biome#4075] [General]
There should be a `requireConfig` or similar option to disable Biome in projects where it is not supposed to run.

Currently it's simply _impossible to progressively migrate_ projects from ESLint and Prettier to Biome, as the Biome extension in VSCode is running on top of all other projects even when no `biome.json` config exists. I'm surprised this is not a feature and is the sole reason I am unable to move my current project into Biome.

**Comment by ematipico** [comment]
This is already addressed in the **preview release** of the VS Code extension

**Comment by ricardobeat** [comment]
Hi @ematipico, can you help me understand what/how it was addressed?

I have switched to the preview release and don't see any new configuration options for the extension. I've also browsed the commit history here and in the `-vscode` repository for the past few weeks and did not find a related feature. Appreciate any pointers.

**Reply by ematipico** [reply]
I suggest you head to the VScode repository, a lot of information is there.

Also, the discord server is another source of information.

There's also a preview of the new docs: https://github.com/biomejs/website/pull/974

Use the VSCode repository to file issues

**Comment by ricardobeat** [comment]
For anyone browsing in the future, this is the relevant change where `requireConfigFile` was added back: https://github.com/biomejs/website/pull/974/files#diff-9a8f442ce68255eeba6efac7da451a7d2d1a0e82e151d29bf66ee27b4cf291a6R80-R87
